ethereum-consensus/src/deneb/polynomial_commitments.rs

```diff
@@ -0,0 +1,138 @@
+#![allow(unused)]
+use crate::{crypto::hash, ssz::ByteVector};
+use alloy_primitives::{uint, U256};
+use blst::min_pk::PublicKey;
+use c_kzg::{Bytes32, Bytes48, Error, KzgSettings};
+use ssz_rs::prelude::*;
```

Comment:
btw once you rebase from `main`, you can do this

```suggestion
use crate::{crypto::hash, ssz::prelude::*};
use alloy_primitives::{uint, U256};
use blst::min_pk::PublicKey;
use c_kzg::{Bytes32, Bytes48, Error, KzgSettings};
```